src/components/NoticeText/BrandRegistration10DlcNoticeText.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+const BrandRegistration10DlcNoticeText: React.FC = () => (
+  <>
+    <p>
+      For the best messaging experience and lowest sending costs, please
+      complete our 10DLC registration form. You are currently on our{" "}
```

Comment:
I would place the pricing plan "You are currently on our..." info on a separate line for readability